src/dict.h

```diff
@@ -181,11 +181,6 @@ typedef struct {
     if ((d)->type->keyDestructor) \
         (d)->type->keyDestructor((d), dictGetKey(entry))
 
-#define dictCompareKeys(d, key1, key2) \
```

Comment:
Hi @moticless I think this function should be removed after https://github.com/redis/redis/pull/13806, i added a new `dictCompareKeys` in `dict.c`

Reply:
Right. Please lmk if you want me to remove it.

Reply:
Thanks for confirmation, i removed it in this PR, `dictCompareKeys` is deprecated, no place to call except this PR, so it is no harm.